<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>785</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1455</width>
    <height>785</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1455</width>
    <height>785</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="xRotateUpButton">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>150</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yRotateUpButton">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>150</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yRotateDownButton">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>210</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xRotateDownButton">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>210</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x-</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>310</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>310</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>280</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>340</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectLineColorButton">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>590</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет линий</string>
    </property>
   </widget>
   <widget class="QSlider" name="selectLineWidth">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>460</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPointColorButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>590</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет точек</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectTypeLinesCBox">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>510</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Сплошная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пунктирная</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="selectTypePointsCBox">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>510</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Круг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="selectPointsSizeSlider">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>460</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selectBGColorButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>550</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет заднего фона</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveScreenButton">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>640</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить скриншот</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="jpegRButton">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>690</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.jpeg</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switcherJpegBmpGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="bmpRButton">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>690</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.bmp</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switcherJpegBmpGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="x_down">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>210</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_down">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>210</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>z-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_up">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>150</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>z+</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="x_animation_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>271</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ось Х</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">animationGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="y_down">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>210</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_up">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>150</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y+</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="y_animation_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>287</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ось Y</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">animationGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="perspective_switch">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>400</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить тип проекции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_up">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>150</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x+</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="stap">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>319</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Остановить</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">animationGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="gifButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>640</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать GIF анимацию</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>256</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Масштаб
модели</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>670</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Формат скриншота:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>440</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Толщина линий</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>440</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Толщина точек</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>490</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип точек</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>490</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип линий</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Смещение по осям</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поворот по осям</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>256</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сдвиг камеры</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>395</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Позиция проекции</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>240</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>620</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSlider" name="perspectivePointPos">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>415</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>130</y>
      <width>20</width>
      <height>116</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>120</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>375</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="zFieldOffsetSB">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>180</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="xFieldOffsetSB">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>180</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yFieldOffsetSB">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>180</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="zRotateDownButton">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>210</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>z-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zRotateUpButton">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>150</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>z+</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yFieldRotateSB">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>180</y>
      <width>55</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="xFieldRotateSB">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>180</y>
      <width>55</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zFieldRotateSB">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>180</y>
      <width>55</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleUpButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>290</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleDownButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>350</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale-</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="scaleSB">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>320</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>255</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Анимация</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>150</number>
    </property>
    <property name="sliderPosition">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>340</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Скорость анимации</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1174</x>
      <y>250</y>
      <width>5</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>250</y>
      <width>5</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="z_animation_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>303</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ось Z</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">animationGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1455</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSave_settings"/>
    <addaction name="actionLoad_settings"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_settings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionLoad_settings">
   <property name="text">
    <string>Load settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="animationGroup"/>
  <buttongroup name="switcherJpegBmpGroup"/>
 </buttongroups>
</ui>
